wit/bindgen/generator.go

```diff
@@ -808,6 +808,13 @@ func (g *generator) flagsRep(file *gen.File, dir wit.Direction, flags *wit.Flags
 
 func (g *generator) enumRep(file *gen.File, dir wit.Direction, e *wit.Enum, goName string) string {
 	var b strings.Builder
+
+	// builder for enum reverse lookup
+	var rb strings.Builder
+	mapName := "map" + goName
+	stringio.Write(&rb, "// ", mapName, " is the mapping of ", goName, " enum to its wit string value,\n", "// which is used for reverse lookup using the String() method.\n")
+	stringio.Write(&rb, "var ", mapName, " = map[", goName, "]string {\n")
```

Comment:
This should be a fixed length Go array, not a map.

Reply:
Since this var is not exported, it doesn't need a doc comment.

Also: doc comments can be wrapped automatically with the doc comment helper.

Reply:
done.